src/pages/markets/static/content/_commodities.js

```diff
@@ -0,0 +1,34 @@
+import { localize } from 'components/localization'
+import FriendlySupport from 'images/svg/friendly-support.svg'
+import MetalAndOil from 'images/svg/precious-metals-and-oils.svg'
+import NoCommission from 'images/svg/no-commission.svg'
+import ResponsiveWebsite from 'images/svg/responsive-website.svg'
+import VariableSpreads from 'images/svg/variable-spreads.svg'
+
+export default [
+    {
+        src: VariableSpreads,
+        alt: 'variable spreads',
+        text: localize('Competitive fixed and variable spreads'),
```

Comment:
can you check if this text is translated, localize function can work when it gets rendered, so putting it into a reserved variables might make it not work, to make it work we can use <Localize instead

```suggestion
        text: <Localize translate_text='Competitive fixed and variable spreads' />,
```

same goes for the others

Reply:
It's not translated. Will check that out 👍🏻